examples/platform/silabs/efr32/wf200/efr_spi.c

```diff
@@ -149,6 +148,12 @@ sl_status_t sl_wfx_host_deinit_bus(void)
  *****************************************************************************/
 sl_status_t sl_wfx_host_spi_cs_assert()
 {
+    configASSERT(spi_sem_sync_hdl);
+    if (xSemaphoreTake(spi_sem_sync_hdl, portMAX_DELAY) != pdTRUE)
+    {
+        return SL_STATUS_TIMEOUT;
```

Comment:
since you use portMAX_DELAY it should never timeout but sure we can have the check